NMDSdevopsServiceAdm/DataEngineering
# Add pir model to estimates

# Description
Add PIR model to estimate selection

# How to test
Unit tests passing
Run in branch: https://eu-west-2.console.aws.amazon.com/states/home?region=eu-west-2#/v2/executions/details/arn:aws:states:eu-west-2:344210435447:execution:add-pir-model-to-estimates-Ind-CQC-Filled-Post-Estimates-Pipeline:86637407-485b-412a-b8be-60ebe3bbf2b9 (Crawler failed because I ran it early to see the data, so was already running)

# Developer checklist
